Cython/Build/IpythonMagic.py

```diff
@@ -106,6 +107,26 @@ def encode_fs(name):
         return name
 
 
+def prepare_captured(captured):
+    captured_bytes = captured and captured().strip()
```

Comment:
Is there a case where `captured` is not callable? Otherwise, I'd like to avoid dead code.
```suggestion
    captured_bytes = captured().strip()
```

Reply:
Here I followed this recipe: https://github.com/cython/cython/blob/3f31af311cdb0df42e0fce283a3667696d2d3668/runtests.py#L1255 and assumed `captured` could be `None` if something went wrong before this line was executed: https://github.com/cython/cython/blob/f15d015941a580f959b7246c7d8b9d9c2aec5109/Cython/Utils.py#L413. but maybe I'm wrong.

Reply:
Ok, right, it's needed. I refactored the code to make it more explicit what gets returned where.